qa/zcash/updatecheck.py

```diff
@@ -381,7 +378,7 @@ def main():
 
     if len(unchecked_dependencies) > 0:
         unchecked_dependencies.sort()
-        print("WARNING: The following dependencies are not being checked for updates by this script: " + ', '.join(unchecked_dependencies))
+        print("WARNING: The following dependencies are not being checked for updates by this script: %s" % ', '.join(unchecked_dependencies))
```

Comment:
Here however, I don't see a benefit. There's no additional readability, and `%s` strings are less efficient than string concatenation: https://stackoverflow.com/questions/38722105/format-strings-vs-concatenation